build/SharedFx.targets

```diff
@@ -38,7 +38,7 @@
 
   <Target Name="BuildSharedFx" DependsOnTargets="ResolveCommitHash">
     <ItemGroup>
-      <_AcceptableSharedFxRIDs Include="win7-x64;win7-x86;osx-x64;linux-x64"/>
+      <_AcceptableSharedFxRIDs Include="win-x64;win-x86;osx-x64;linux-x64"/>
```

Comment:
This is going to require a change in the build configuration on the CI.

Reply:
That's fine. I can update it.